android/filament-utils-android/src/main/java/com/google/android/filament/utils/ModelViewer.kt

```diff
@@ -284,7 +288,7 @@ class ModelViewer(
      * @param frameTimeNanos time in nanoseconds when the frame started being rendered,
      *                       typically comes from {@link android.view.Choreographer.FrameCallback}
      */
-    fun render(frameTimeNanos: Long) {
+    fun render(frameTimeNanos: Long, callback: ((Bitmap) -> Unit)? = null) {
```

Comment:
This is essentially an public API so two questions:
- do we want to add this callback in a public API?
- this needs to be documented in the javadoc above

Reply:
what I don't like about this API is that the user doesn't control the format of the bitmap, nor the region captured.

Format: what if the rendering is HDR?

It seems like a debug API, but it's right there on the main render() API.

Maybe a ModelViewer.debugGetNextFrameCallback()? Or, if we can make the API more generic?

Reply:
done